src/walletConnect/saga.ts

```diff
@@ -476,6 +476,7 @@ function* showActionRequest(request: Web3WalletTypes.EventArguments['session_req
         feeCurrencies,
         decreasedAmountGasFeeMultiplier: 1,
         baseTransactions: [normalizedTx],
+        origin: 'wallet-connect' as const,
```

Comment:
why the 'as const' here?

Reply:
sagas + TS, type is inferred as `string`